README.md

```diff
@@ -5,33 +5,82 @@
   <img src="./.github/assets/topos_logo_dark.png#gh-dark-mode-only" alt="Logo" width="200">
   <br />
   <p align="center">
-  <b>Topos Playground</b> is a CLI make it simple to run a local Topos devnet 🚀
+  <b>Topos Playground</b> is a CLI to make it simple to run a local Topos devnet 🚀
   </p>
   <br />
 </div>
 
 ## Getting Started
 
+The Topos Playground is a CLI that allows you to easily run and manage a local Topos devnet. It depends on Docker and NodeJS, and manages a number of components in order to locally run multiple subnets, multiple TCE nodes, and a dApp with supporting infrastructure, capable of cross-subnet messaging.
+
 ### Requirements
 
+- [Docker](https://docs.docker.com/get-docker/_) version 17.06.0 or greater
```

Comment:
Should mention `docker-compose` as well. At least on my machine it was not installed causing a somewhat puzzling error of `info: unknown shorthand flag: 'd' in -d`. I'm on macOS and I have installed Docker with homebrew.

Reply:
Good suggestion. I'll follow up on it after checking whether I forgot about installing docker compose on my laptop, or whether the typical linux install bundles it (I am betting that I just forgot that I did that, as I installed docker quite a while ago).

Reply:
@wyhaines resolving this, we can fix it in the next iteration perhaps?